io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/TypeSelectionPage.java

```diff
@@ -106,16 +107,20 @@ public void selectionChanged(SelectionChangedEvent event) {
 				if (sel.getSelectedObject() != null) {
 					TypeDefinition type = sel.getSelectedObject();
 					CSVConfiguration conf = type.getConstraint(CSVConfiguration.class);
-					Boolean skip = conf.skipFirst();
-					button.setSelection(skip);
-					label.getParent().layout();
+					int skipNlines = conf.skipNlines();
+					text.setSelection(skipNlines);
+					label1.getParent().layout();
 				}
 			}
 		});
 
-		button = new Button(page, SWT.CHECK);
-		button.setText("Skip first line");
-		button.setSelection(true);
+		// button = new Button(page, SWT.CHECK);
+		// button.setText("Skip first line");
+		// button.setSelection(true);
+
+		label2 = new Label(page, SWT.NONE);
+		label2.setText("No. of lines to skip");
+		text = new Text(page, SWT.SINGLE | SWT.BORDER);
```

Comment:
Would it maybe make sense to use a component that only allows numbers as input?
For instance `org.eclipse.swt.widgets.Spinner` - there are a couple of examples in the hale studio code.